packages/nextjs-webpack-plugin/README.md

```diff
@@ -33,7 +33,62 @@ Using pnpm:
 pnpm add @codecov/nextjs-webpack-plugin --save-dev
 ```
 
-## Example
+## Public Repo Example - GitHub Actions
+
+This configuration will automatically upload the bundle analysis to Codecov for public repositories. When an internal PR is created it will use the Codecov token set in your secrets, and if running from a forked PR, it will use the tokenless setting automatically. For setups not using GitHub Actions see the following [example](#public-repo-example---non-github-actions). For private repositories see the following [example](#private-repo-example).
+
+```typescript
+// next.config.mjs
+import { codecovNextJSWebpackPlugin } from "@codecov/nextjs-webpack-plugin";
+
+export default {
+  webpack: (config, options) => {
+    config.plugins.push(
+      codecovNextJSWebpackPlugin({
+        enableBundleAnalysis: true,
+        bundleName: "example-nextjs-webpack-bundle",
+        uploadToken: process.env.CODECOV_TOKEN,
```

Comment:
Same thing here re: whether it makes sense in the public repo example to omit the `uploadToken` since it's technically not required